addon/modifiers/has-scrolled.js

```diff
@@ -64,23 +64,23 @@ export default class ScrollPositionModifier extends Modifier {
   }
 
   #calculateScrollPosition() {
-    const isAtScrollStart = this.#isAtScrollStart(this.element);
+    const hasScrolled = this.#hasScrolled(this.element);
     const isAtScrollEnd = this.#isAtScrollEnd(this.element);
+    let hasRemainingScroll = true;
```

Comment:
I think this can be simplified to:
```suggestion
  const hasRemainingScroll = !isAtScrollEnd;
```

With the `if` below being removed.

Reply:
That's a god catch ➕